tensorflow_recommenders_addons/dynamic_embedding/python/ops/dynamic_embedding_variable.py

```diff
@@ -233,8 +233,7 @@ def _insert_de_shard_from_file_system(
   _values_tensor_dataset = readers.FixedLengthRecordDataset(
       shard_values_file_list,
       record_bytes=_value_dtype.size * value_dim,
-      buffer_size=buffer_size * value_dim).padded_batch(insert_num_once *
-                                                        value_dim,
```

Comment:
Why here need to delete `value_dim`?

Reply:
_values_tensor_dataset record_bytes is already _value_dtype.size * value_dim, if we padded_batch the bytes, they could be the size _value_dtype.size * value_dim * insert_num_once * value_dim. When they are reshaped, their shape would not compatible with _keys_tensor_dataset.